_commands/scripting/dollarUnderscore.md

```diff
@@ -0,0 +1,41 @@
+---
+---
+
+$\_
+-----
+
+`$_` is a special shell variable which always holds the last argument of the most recent command.
+
+~~~ bash
+$ touch example.txt
+$ echo $_
+example.txt
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+$\_ is a variable, not a command so its main use is to save you some typing.
```

Comment:
This is a little inconsistent from above.
I recommend you change the first few characters for line 19 to look like line 7.